runtime/service/npud/core/Server.h

```diff
@@ -43,13 +44,16 @@ class Server
     return server;
   }
 
+  static std::shared_ptr<Core> &core(void) { return instance()._core; }
+
 private:
   Server() noexcept;
 
```

Comment:
```suggestion
        Server(const Server &);
        void operator=(const Server &);
```
or
```suggestion
public:
        Server(const Server &) = delete;
        void operator=(const Server &) = delete;

```

Reply:
This change is not related to this PR. I will make a new PR for this.

Reply:
OK. But I think it would be better to add disabling copy constructor and operator when using singleton pattern.

Reply:
Related PR: #9995